contracts/exchange/contracts/src/MixinExchangeRichErrors.sol

```diff
@@ -28,6 +28,17 @@ contract MixinExchangeRichErrors is
     IExchangeRichErrors
 {
     // solhint-disable func-name-mixedcase
+    function BatchMatchOrdersError(BatchMatchOrdersErrorCodes errorCode)
+        internal
+        pure
+        returns (bytes memory)
+    {
+        abi.encodeWithSelector(
```

Comment:
Can we add an explicit return here? I didn't even know you could implicitly return unnamed variables.

Reply:
I didn't either... That's pretty weird. I'll update!